micrometer/pom.xml

```diff
@@ -27,6 +27,7 @@
 
   <properties>
     <main.basedir>${project.basedir}/..</main.basedir>
+    <micrometer.version>1.10.0-SNAPSHOT</micrometer.version>
```

Comment:
That is going to be a problem... will only be able to merge after 1.10 is released

Reply:
Is 1.10 really needed?

Reply:
I guess this is temporary to be able to compile and run tests.
1.10 is needed for this since the new Observation API is in that release.

Reply:
Yeah this PR would be merge once Micrometer 1.10.0 is out. Then there would be no SNAPSHOT dependencies.